lib/improver/tests/acceptance/test_blend_adjacent_points.py

```diff
@@ -45,8 +45,10 @@ def test_basic_mean(tmp_path):
     kgo_path = kgo_dir / "kgo.nc"
     multi_prob = sorted(kgo_dir.glob("multiple_probabilities_rain_*H.nc"))
     output_path = tmp_path / "output.nc"
-    args = ["forecast_period", "2", "--units", "hours", "--width", "3.0",
-            *multi_prob, output_path]
+    args = ["--coordinate", "forecast_period", "--central-point", "2",
+            "--units", "hours", "--width", "3.0",
```

Comment:
Stylistic suggestion: new line per argument, ie:
```
args = ["--coordinate", "forecast_period",
        "--central-point", "2",
        "--width", "3",
        "--units", "hours",
```
And would be nice to be consistent with `2` vs `3.0` - I don't mind which, but the CLI will interpret both as floats so it's not sensitive to the detail here.

Reply:
Agreed on style - happy to change. (I just took what was there)

Reply:
Done. 😄